v2l/channel.go

```diff
@@ -46,18 +152,58 @@ func CreateChannel(server, chName, contentTemplatePath string, gopDurMS, nrGopsP
 	if err != nil {
 		return nil, err
 	}
-	err = json.Unmarshal(respBody, &schedule)
+
+	var serverSchedule Schedule
+	err = json.Unmarshal(respBody, &serverSchedule)
 	if err != nil {
 		return nil, err
 	}
-	printJSON("schedule", &schedule)
-	channel.Schedule = &schedule
-	channel.LastSCTEEventID = 1
+	log.Printf("Channel: %s, Schedule entries: %d", chName, len(serverSchedule.Entries))
+	channel.Schedule = &serverSchedule
+	if channel.LastSCTEEventID != serverSchedule.LastSCTEEventID() {
+		log.Printf("Expected LastSCTEEventID %d, vod2cbm returned: %d [channel: %s]",
+			channel.LastSCTEEventID,
+			serverSchedule.LastSCTEEventID(),
+			chName)
+	}
 	return &channel, nil
 }
 
-// nowToSegNr - calculate what segment was last produced
-func nowToSegNr(gopDurMS, nrGopsPerSegment int64) int64 {
-	nowMS := time.Now().UnixNano() / 1_000_000
-	return nowMS/(gopDurMS*nrGopsPerSegment) - 1
+// CreateChannels - create a slice filled with channels
+func CreateChannels(nofChannels int,
```

Comment:
Use "nrOfChannels" instead of "nofChannels" to be consistent with the rest of the names.